packages/relay/src/lib/dispatcher/rpcMethodDispatcher.ts

```diff
@@ -216,4 +226,35 @@ export class RpcMethodDispatcher {
       requestId,
     );
   }
+
+  /**
+   * Maps a MirrorNodeClientError to a standardized JsonRpcError.
+   *
+   *
+   * @param error - The MirrorNodeClientError instance to map to a JsonRpcError.
+   * @returns A JsonRpcError representing the mapped Mirror Node error, including
+   *          the appropriate error code and message.
+   */
+  private mapMirrorNodeError(error: MirrorNodeClientError): JsonRpcError {
```

Comment:
Wouldn't it be better for the `MirrorNodeClientError` class to be responsible for its own transformation and have this method there?